modules/cram/templates/spectre_call.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -euo pipefail
+
+mosdepth () {
+    local args=()
+    args+=("--threads"  "!{task.cpus}")
+    args+=("--by" "1000")
+    args+=("--fasta" "!{paramReference}")
+    args+=("--mapq" "20")
+    args+=("--fast-mode")
+    args+=("mosdepth")
+    args+=("!{cram}")
+    ${CMD_MOSDEPTH} "${args[@]}"
+}
+
+call_copy_number_variation () {
+    local args=()
+    args+=("CNVCaller")
+    args+=("--coverage" "mosdepth.regions.bed.gz")
+    args+=("--sample-id" "!{sampleId}")
+    args+=("--output-dir" "spectre")
+    args+=("--reference" "!{paramReference}")
+    args+=("--metadata" "!{paramMetadata}")
+    args+=("--blacklist" "!{paramBlacklist}")
+    if [ "!{sampleSex}" == "male" ]; then
+        args+=("--ploidy-chr" "chrX:1,chrY:1")
+    else
+        args+=("--ploidy-chr" "chrX:2")
+    fi
+
+    ${CMD_SPECTRE} "${args[@]}"
+}
+
+postprocess() {
+    # empty result of spectre is a vcf file and not a vcf.gz. https://github.com/fritzsedlazeck/Spectre/issues/26
```

Comment:
issue has been fixed, we could consider upgrading to a patched version. alternatively remove this workaround after the next release. what do you think?